src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -387,9 +389,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthorization(options =>
             {
-                //We need this policy because it is a single way to implicitly use the two schema (JwtBearer and ApiKey)  authentication for resource based authorization.
+                //We need this policy because it is a single way to implicitly use the three schemas (JwtBearer, ApiKey and Basic) authentication for resource based authorization.
```

Comment:
could you also  add a new schema here please https://github.com/VirtoCommerce/vc-platform/blob/dev/src/VirtoCommerce.Platform.Web/Security/Authorization/PermissionAuthorizationPolicyProvider.cs#L50

Reply:
thanks for noticing that. added.